finagle-http/src/test/scala/com/twitter/finagle/http/StatusTest.scala

```diff
@@ -15,4 +16,45 @@ class StatusTest extends FunSuite {
     assert(Status(599).reason == "Server Error")
     assert(Status(601).reason == "Unknown Status")
   }
+
+  test("Pattern match usability check") {
```

Comment:
can you break this up into separate tests for each thing you're testing?  ie, "matches unknown" "matches informational" etc.